Source/Fuse.Reactive.JavaScript/ModuleInstance.uno

```diff
@@ -116,4 +122,4 @@ namespace Fuse.Reactive
 			}
 		}
 	}
-}
```

Comment:
Nit: unnecessary whitespace change

Reply:
It just makes sure the file has a trailing new-line, which is required by the unix-standard to be considered a text-file.

I usually just fix these as I encounter them, but I can separate them out into a separate commit if that helps?

Reply:
Right, ok. I think it's fine as-is, but it looks like you already added them as separate commits, which I do think is a bit cleaner :)